getting-started/scaffolding.md

```diff
@@ -33,7 +33,7 @@ Next we'll be prompted to support [CORs](https://developer.mozilla.org/en-US/doc
 
 Now let's choose our database. You can see that Feathers supports a bunch through the generator and even more outside the generator. Let's use the default [NeDB](https://github.com/louischatriot/nedb). NeDB is a local file-system based database so we don't have to start a separate database server. You can find out more about our database adapters in the [Databases chapter](../databases/readme.md).
 
-Since pretty much every app needs authentication of some sort we generate almost everything you need to get going. You can learn more about authentication in Feathers by reading the [Authentication chapter](../../authentication/readme.md). In our case we will just use the default local authentication.
```

Comment:
I think it should actually be `../authentication/readme.md`. Gitbook automatically converts those to html files for us. If the path you are linking to doesn't exist then Gitbook won't convert it. By making it `.html` breaks navigation in the feathers-docs github repo. If you want to remove the extra `..` and convert back to `.md` that would be sweet and we can merge this. Thanks!